### PR TITLE
fix: cleaning warning messages

### PR DESCRIPTION
**Pull request by Gusinuhe**
### Description

Cleaning of warning messages 

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### CODE REVIEW

File: Runtime/Interaction/ControllerManager.cs [hunk 1]
```diff
@@ -246,39 +246,44 @@ public bool IsControllerInteracting(ControllerStates controller)
...
 
         [SerializeField]
         [Tooltip("The buttons on the controller that will trigger a transition to the Teleport Controller.")]
+#pragma warning disable CS0649 // Disable "field is never asigned" warning, it is assigned from the Unity Editor.
         private Button teleportButton;
+#pragma warning restore CS0649
 
         [SerializeField]
         [Tooltip("The buttons on the controller that will force a deactivation of the teleport option.")]
+#pragma warning disable CS0649 // Disable "field is never asigned" warning, it is assigned from the Unity Editor.
```
Comment (aleksei-korolev) [inline Runtime/Interaction/ControllerManager.cs:270]: Simply assign null instead?

Comment (Gusinuhe) [reply Runtime/Interaction/ControllerManager.cs:270]: `Button` is a strut and can't be null.

**Comment by SimonTheSourcerer** [comment]
This is not a bugfix, this is probably a chore or a refactor :P

File: Runtime/Interaction/ControllerManager.cs (hunk 1 above)
Comment (aleksei-korolev) [reply Runtime/Interaction/ControllerManager.cs:270]: default(Button)

**Pull request merged**